Library/Homebrew/test/cmd/info_spec.rb

```diff
@@ -18,6 +18,14 @@
       .and not_to_output.to_stderr
       .and be_a_success
   end
+
+  it "looks for a Cask with the same name if a given Formula does not exist", :needs_network do
+    setup_remote_tap "homebrew/cask"
+
+    expect { brew "info", "firefox" }
+      .to output(/No available formula with the name "firefox"/).to_stderr
+      .and output(/firefox/).to_stdout
```

Comment:
Can you make this check more specific to the cask output?

Reply:
Should I use a different cask besides firefox? I used it because it was used in similar integration tests in `search_spec.rb`, but if I make the test match the exact output I would need to use a regex or something to match the version number, which isn’t something that I’ve seen done in other tests.

Reply:
even matching e.g. `Mozilla Firefox` or `firefox: ` or the URL would be preferable.